### PR TITLE
feat: add mullvad-vpn recipe and builds

### PR DESCRIPTION
**Pull request by tulilirockz**
Should work just fine, it also moves the program files from /opt to /usr

### CODE REVIEW

File: mullvad-vpn/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# docker-ce
```
Comment (cverna) [inline mullvad-vpn/README.md:1]: This needs to be updated :-)

Comment (tulilirockz) [reply mullvad-vpn/README.md:1]: omg true i forgot to update it!

File: mullvad-vpn/Containerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    && \
+    mv "/opt/Mullvad VPN" "/usr/lib/Mullvad VPN" \
+    && \
+    ln -sf "/usr/lib/Mullvad VPN/mullvad-vpn" /usr/bin/mullvad-vpn \
+    && \
+    ln -sf "/usr/lib/Mullvad VPN/mullvad-gui" /usr/bin/mullvad-gui \
+    && \
+    chmod 4755 "/usr/lib/Mullvad VPN/chrome-sandbox" \
+    && \
+    sed -i 's|^Exec=/opt/Mullvad VPN|Exec=/usr/bin|g' /usr/share/applications/1password.desktop \
```
Comment (travier) [inline mullvad-vpn/Containerfile:19]:
```suggestion
    sed -i 's|^Exec=/opt/Mullvad VPN|Exec=/usr/bin|g' /usr/share/applications/mullvad-vpn.desktop \
```

**Review by travier — APPROVED** [review]
Thanks. LGTM

**Pull request merged**